.github/workflows/im-reusable-finish-build-workflow.yml

```diff
@@ -1,168 +1,217 @@
+#        ]
+#      ms-teams-uri: ${{ vars.MS_TEAMS_URI }}  # Use this input (preferred) or the secret
+
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        description: 'The runner that this workflow will run on.'
+        required: false
+        type: string
+        default: 'im-linux'
+      next-version:
+        description: The next generated version
+        type: string
+        required: false
+        default: 'N/A'
+      title-of-teams-post:
+        description: 'Title of the Teams post that reports the build status.'
+        required: true
+        type: string
+      is-merge-to-main:
+        description: Flag indicating whether this is a merge to the default branch. Expected `true` or `false`.
+        required: true
+        type: string
+      timezone:
+        description: 'Timezone for the project.  Defaults to america/denver.'
+        required: false
+        type: string
+        default: 'america/denver'
+      additional-conclusions:
+        description: Valid JSON array of additional conclusion items that should be used to determine if the workflow succeeded or failed
+        required: false
+        type: string
+        default: ''
+      additional-pr-comment-content:
+        description: Additional PR comment content aside from workflow run and next version.
+        required: false
+        type: string
+      custom-facts-for-team-channel:
+        description: The custom facts that will be included in the post in the team's channel.  By default Workflow, Run, Actor and Version are included.
+        required: false
+        type: string
+      ms-teams-uri:
+        description: The URI for the teams channel where a status will be posted.  Either this value or the secret MS_TEAMS_URI must be provided.  This input is the preferred way to provide the URI but the secret should be used instead if the value is defined as a secret.
+        required: false
+        type: string
+
+    secrets:
+      MS_TEAMS_URI:
+        description: The URI for the teams channel where a status will be posted.  Either this value or the input ms-teams-uri must be provided.  The input is the preferred way to provide the URI but the secret should be used if the value is defined as a secret.
+        required: false
+
+jobs:
+  finish-build:
+    runs-on: ${{ inputs.runs-on }}
+
+    steps:
+      - name: Check for missing inputs
+        uses: actions/github-script@v7
+        with:
+          script: |
+            // Some teams have these as a secret and some as a var, allow both ways but check at least one is present
+            const teamsUri = '${{ inputs.ms-teams-uri || secrets.MS_TEAMS_URI }}';
+            if (!teamsUri || teamsUri.trim().length === 0){
+              core.setFailed('The MS_TEAMS_URI secret or the ms-teams-uri input (preferred) must be provided.');
+            }
+
+      - name: Print inputs

```

Comment:
Do the inputs of the reusable workflow get printed in the log like when actions are called?

Reply:
I think there are two use cases for the extra step.  First, it shows you what the action is going to use for a default if one wasn't provided which is kind of nice.  In the example below the tag-prefix isn't provided and you can see how the built in Inputs shows it versus the Print Inputs step.
![image](https://github.com/im-practices/.github/assets/2112531/c6c11d78-b2aa-4bec-9569-0443aa4bf57f)


The second use case is the built in inputs doesn't print whether it got secrets or not, which is what prompted me to do this in the first place.  Obviously the new step doesn't print the value, but it does show whether it received a value or not for the secret.
![image](https://github.com/im-practices/.github/assets/2112531/b56fcf16-d998-4eea-8c79-d485dd38ac1c)